### PR TITLE
Proposal for different lineJustify small fixes

### PR DESCRIPTION
**Pull request by jilster**
Worked like a charm after changing styleMap to _styleMap. I tested with Textbox, Text, IText and export to SVG

### CODE REVIEW

File: src/shapes/text.class.js [hunk 1]
```diff
@@ -148,7 +148,8 @@
     linethrough:       false,
 
     /**
-     * Text alignment. Possible values: "left", "center", "right" or "justify".
+     * Text alignment. Possible values: "left", "center", "right", "justify",
+     * "justify-center" or "justify-right".
```
Comment (asturur) [inline src/shapes/text.class.js]: and justify-left too.

**Comment by asturur** [comment]
aren't we missing some pieces? i committed more on the other side.

**Comment by asturur** [comment]
ohhh this is fixing mine :D sorry i 'm busy doing other stuff.
Let's put this in and continue

File: src/shapes/text.class.js (hunk 1 above)
Comment (jilster) [reply src/shapes/text.class.js]: I now realise what you meant here. I thought we were going to overwrite the behaviour of "justify", but we are keeping that one intact and are adding "justify-left". Are you going to change this in https://github.com/kangax/fabric.js/pull/4437 or should i make another PR?

**Comment by redlive** [comment]
Is there a way to make an alignment described above per line of text as well? When I say the line I mean text split by `\n`

**Pull request merged**